util/src/main/java/io/kubernetes/client/util/ModelMapper.java

```diff
@@ -325,4 +362,22 @@ private static void initModelMap() throws IOException {
         .findFirst()
         .orElse(new MutablePair(null, name));
   }
+
+  static class BiDirectionalMap<K, V> {
+    private Map<K, V> kvMap = new HashMap<>();
```

Comment:
I think we want this to be `ConcurrentHashMap<>()` ?

Reply:
the getter/setter of the bi-directional-map are locked by `synchronized` (also to keep the consistency between the two maps) so a plain hash-map is sufficient, isn't it?